src/server/CMakeLists.txt

```diff
@@ -4,5 +4,6 @@ set(server_SRCS
 	${CMAKE_CURRENT_SOURCE_DIR}/mods.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/player_sao.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/serveractiveobject.cpp
+	${CMAKE_CURRENT_SOURCE_DIR}/serverinventorymgr.cpp
```

Comment:
What about Android?

Reply:
android is not using those cmakelists ? i should update :D

Reply:
`$(wildcard ../../../src/server/*.cpp)           \`

The new android build is <3 :)